docs/dependency-specification.md

```diff
@@ -161,6 +161,20 @@ poetry config http-basic.git-org-project username token
 poetry add git+https://github.com/org/project.git
 ```
 
+In cases where the package you want to install is located in a subdirectory of the VCS repo, you can use the subdirectory option:
```

Comment:
This whole added section should be moved higher (line 147 maybe?)  since now it interferes with information about git configuration.

Reply:
Agreed! Changes it accordingly. Thanks for the review